packages/babel-helper-explode-assignable-expression/src/index.ts

```diff
@@ -37,8 +45,17 @@ function getObjRef(node, nodes, file, scope) {
   return temp;
 }
 
-function getPropRef(node, nodes, file, scope) {
+function getPropRef(
+  node: MemberExpression,
+  nodes: Array<AssignmentExpression>,
+  scope: Scope,
+): Identifier | Literal {
   const prop = node.property;
+  if (t.isPrivateName(prop)) {
+    throw new Error(
+      "We can't generate property ref for private name, please install `@babel/plugin-proposal-class-properties`",
+    );
```

Comment:
We already throws when private name is in property ([REPL](https://babel.dev/repl#?browsers=chrome%2085&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAECC0G8BQ1oGIAO0C80CMA3CtALYAUAlIsagC4AWAlhAHSbQBUHuhxAvkj5A&debug=false&forceAllTransforms=false&shippedProposals=true&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env&prettier=false&targets=&version=7.12.9&externalPlugins=%40babel%2Fplugin-proposal-optional-chaining%407.12.7%2C%40babel%2Fplugin-syntax-class-properties%407.12.1%2C%40babel%2Fplugin-transform-exponentiation-operator%407.12.1))
```
Property right of AssignmentExpression expected node to be of a type ["Expression"] but instead got "PrivateName"
```

Now we throws polished error message.